client/stories/SimpleMultiCheckboxDropdown.stories.js

```diff
@@ -1,41 +0,0 @@
-import React, { useState } from "react"
-import SimpleMultiCheckboxDropdown from "../src/components/SimpleMultiCheckboxDropdown"
```

Comment:
This story was testing an ANET component; do we want to drop it?

Reply:
@VassilIordanov created a separate branch for the issue #3838. I took all of his modifications into this branch which yields in removal of some components from the storybook if I'm not mistaken.

Reply:
Let's keep all storybooks related to ANET components for now. If we want to remove them at some point, we can always do that later. So please restore this file, @asafkaganbezgin